src/createBrowserApp.js

```diff
@@ -82,7 +82,9 @@ export default function createBrowserApp(App) {
       const activeNav = this._navigation.getChildNavigation(childKey);
       const opts = App.router.getScreenOptions(activeNav);
       this._title = opts.title || opts.headerTitle;
-      document.title = this._title;
+      if (this._title) {
```

Comment:
Shouldn't we check for `undefined`? What if the user wants to set the title to an empty string or number?